apps/nrm/models/submission.js

```diff
@@ -0,0 +1,183 @@
+'use strict';
+
+/* eslint complexity: 0 max-statements: 0 */
+
+module.exports = data => {
+  const response = {};
+
+  response.Type = '560734';
```

Comment:
just out of curiosity not a request for change
erm what is this number? should it be in config?

Reply:
it appears that's the type of request. I'd say we need to clarify why this is needed then we can work out where it should live for now we can put it in and refactor once we know